hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```diff
@@ -17359,7 +17359,8 @@ objects:
               message: Prometheus could not scrape fluentd {{ $labels.instance }}
                 for more than 10m.
               summary: Fluentd cannot be scraped
-            expr: 'absent(up{job="collector"} == 1)
+            expr: 'absent(up{job="collector"} == 1) and absent(up{job="fluentd"} ==
+              1)
```

Comment:
is there a reason this needs to be a new line?

Reply:
These were auto generated from running make. This also happened with other expressions in the files.

Reply:
Gotcha, thanks!